Web.md

```diff
@@ -124,6 +124,81 @@ MIME type mismatch or `opaque` response types
 [reject](http://tc39.github.io/ecma262/#sec-rejectpromise) the Promise with a
 `WebAssembly.CompileError`.
 
+### Structured Clone of a `WebAssembly.Memory`
+
+A `WebAssembly.Memory` is under some circumstances a
```

Comment:
Right, it's always a serializable object. But sometimes that serialization throws.